lib/generators/react_on_rails/base_generator.rb

```diff
@@ -99,7 +99,7 @@ def add_yarn_dependencies
         css_minimizer_webpack_plugin_version = node_major_version >= 18 ? "" : "@<6.0.0"
         css_minimizer_webpack_plugin = "css-minimizer-webpack-plugin#{css_minimizer_webpack_plugin_version}"
 
-        run "yarn add 'css-loader@<7.0.0' '#{css_minimizer_webpack_plugin}' mini-css-extract-plugin style-loader"
+        run "yarn add 'css-loader@<7.0.0' '#{css_minimizer_webpack_plugin}' mini-css-extract-plugin 'style-loader@<4.0.0'" # rubocop:disable Layout/LineLength
```

Comment:
I don't see why we're locking this:

https://github.com/webpack-contrib/style-loader/releases

New projects can update or change the dependency if needed.